.github/workflows/codeql-analysis.yml

```diff
@@ -1,49 +1,47 @@
 name: "Code scanning - action"
 
 on:
-  push:
-    branches: [master, ]
-  pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [master]
-  schedule:
-    - cron: '0 22 * * 5'
+    push:
+        branches: [master]
+    pull_request:
+        # The branches below must be a subset of the branches above
+        branches: [master]
```

Comment:
[`.editorconfig` says](https://github.com/kovidgoyal/kitty/blob/master/.editorconfig#L5)

```
indent_size = 4
```

That's likely what's causing the spacing `diff` here.